src/diffusers/utils/hub_utils.py

```diff
@@ -393,6 +399,119 @@ def _get_model_file(
             )
 
 
+# Taken from https://github.com/huggingface/transformers/blob/main/src/transformers/utils/hub.py
```

Comment:
It's adaped from `transformers`. Differs quite heavily actually.

Reply:
It could be helpful to add a comment about how this differs. If the difference is contained to a few lines, the comment could be added there, otherwise a more general comment at the start of the function would be nice.

Reply:
Resolved.